muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/base/material-loader/fields/better-math-field/toolbar/index.tsx

```diff
@@ -86,7 +86,7 @@ export default class MathFieldToolbar extends React.Component<
           (s as SpecialCharacterType).latexCommand ||
           (s as SpecialCharacterType).character,
         html: (s as SpecialCharacterType).character,
-        useWrite: true,
+        useWrite: (s as SpecialCharacterType).noWrite ? false : true,
```

Comment:
If useWrite is expect to be type boolean and it is always present, then there is no need to use ternary.